src/models.js

```diff
@@ -469,10 +469,17 @@ module.exports = ({ cooler, isPublic }) => {
           }
         }
 
+        const isPost =
+          lodash.get(msg, "value.content.type") === "post" &&
+          lodash.get(msg, "value.content.text", false) !== false;
+        const hasRoot = lodash.get(msg, "value.content.root", false) !== false;
+        const hasFork = lodash.get(msg, "value.content.fork", false) !== false;
+
         const channel = lodash.get(msg, "value.content.channel");
         const hasChannel = typeof channel === "string" && channel.length > 2;
+        const isRoot = hasRoot === false;
 
-        if (hasChannel) {
+        if (hasChannel && isRoot) {
```

Comment:
does this change behaviour?

Reply:
Yes, this bit from the PR description is relevant here:

> This branch has types for unmerged pull requests (#339 and #341), so I'm waiting until someone has time to merge those before this PR is ready for review.

This code is from #341, which makes it so that we only display the `channel` on root posts that set the channel, not on every reply to that post.